src/api/app/views/webui/package/_buildstatus.html.haml

```diff
@@ -8,39 +8,38 @@
 
 - buildresults.results.each_pair do |package, results|
   %h5.text-primary.mt-3.mb-3= package
-  .mb-2#package-buildstatus
-    .px-3
-      - if results.present?
-        - previous_repo = nil
-        - results.each do |result|
-          :ruby
-            repository_name = result.repository.tr('.', '_')
-            package_name = package.tr('.:', '_')
-            expanded = collapsed_repositories[package_name] ? collapsed_repositories[package_name].exclude?(repository_name) : true
-          - if result.repository != previous_repo
-            .row.py-1.bg-light
-              .col{ title: result.repository.to_s }
-                = link_to(word_break(result.repository, 22),
-                  package_binaries_path(project: project, package: package, repository: result.repository),
-                  data: { content: "Binaries for #{result.repository}", placement: 'top', toggle: 'popover' })
-                = link_to('#', aria: { controls: "collapse-#{package_name}-#{repository_name}", expanded: expanded }, class: 'ml-1',
-                          data: { toggle: 'collapse' }, href: ".collapse-#{package_name}-#{repository_name}", role: 'button') do
-                  %i.fas.fa-chevron-left.expander{ title: 'Show build results for this repository' }
-                  %i.fas.fa-chevron-down.collapser{ title: 'Hide build results for this repository' }
+  #package-buildstatus
+    - if results.present?
+      - previous_repo = nil
+      - results.each do |result|
+        :ruby
+          repository_name = result.repository.tr('.', '_')
+          package_name = package.tr('.:', '_')
+          expanded = collapsed_repositories[package_name] ? collapsed_repositories[package_name].exclude?(repository_name) : true
```

Comment:
Would be awesome that this ruby block could live in a helper. What do you think?

Reply:
Done. Please have a look again.